Packs/Okta/ReleaseNotes/3_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### New: IAMInitOktaUser
+- Generates a password and sets an Okta user account with that password, enables the account and sends an email to the user with the account information.
```

Comment:
don't forget to update here the description too (in addition to the yml as discussed)